packages/kbn-unified-data-table/src/components/data_table_columns.tsx

```diff
@@ -145,7 +145,10 @@ function buildEuiGridColumn({
   const column: EuiDataGridColumn = {
     id: columnName,
     schema: getSchemaByKbnType(columnType),
-    isSortable: isSortEnabled && (isPlainRecord || dataViewField?.sortable === true),
+    isSortable:
+      isSortEnabled &&
+      // TODO: would be great to have something like `sortable` flag for text based columns too
```

Comment:
just to be sure, you mean sortable for the UI, and not ES, right, so we would need to check for less conditions, right